src/icui/lib/accessibility.ts

```
@@ -0,0 +1,453 @@
+            } else {
+              newIndex = prev;
+            }
+          }
+          
+          return Math.min(items.length - 1, newIndex);
+        });
+        break;
+
+      case 'Home':
+        event.preventDefault();
+        let firstEnabled = 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)